src/libsync/propagateuploadng.cpp

```diff
@@ -316,7 +478,7 @@ void PropagateUploadFileNG::startNextChunk()
     }
 
     QMap<QByteArray, QByteArray> headers;
-    headers["OC-Chunk-Offset"] = QByteArray::number(_sent);
+    // headers["OC-Chunk-Offset"] = QByteArray::number(_sent);
```

Comment:
What about this? (backwards compat)

Reply:
Yeah I `grep`d core for `OC-Chunk-Offset` and found nothing, didn't even think about legacy. Will uncomment.

Reply:
@guruz I just tested v9.1 and it seems that it's not even using this path, it's using v1 ...

Reply:
It was probably specified when the protocol was drafted but ended up not being used by the oC server. Other servers might very well use it though. The spec says 

> The client sends an additional header with every chunk: OC-Chunk-Offset is the starting position of the chunk in the final files, in bytes.

It should be straightforward to do exactly that, right?

Reply:
Yeah, that would just be `_currentChunk` now, done.